### PR TITLE
contiv k8s container now tagged with git version

### PR DESCRIPTION
**Pull request by chrisplo**
the container build can leverage other new tasks to simplify the build
script considerably

drive-by: found the release notes for the container release was wrong

Signed-off-by: Chris Plock <chrisplo@cisco.com>

### CODE REVIEW

**Comment by chrisplo** [comment]
build PR

**Comment by dseevr** [comment]
build pr

**Comment by unclejack** [comment]
`make host-build-docker-image` is failing for me with this branch:
```
Successfully built 0594eb131267
Successfully tagged netplugin-build:1.1.0-rc.1-400-ga0a08e2
docker rm netplugin-build 2>/dev/null || :
c_id=$(docker create --name netplugin-build netplugin-build:1.1.0-rc.1-400-ga0a08e2) && \
for f in netplugin netmaster netctl contivk8s netcontiv; do \
        docker cp ${c_id}:/go/bin/$f bin/$f; done && \
docker rm ${c_id}
bf3638a1e74e0d196b4f2274f863eac1aeeb179d2990107d58c7c08d0db5a62c
tar: scripts/netcontain/bin: Cannot open: No such file or directory
tar: Error is not recoverable: exiting now
Makefile:257: recipe for target 'host-build-docker-image' failed
make: *** [host-build-docker-image] Error 2

```

**Comment by unclejack** [comment]
build PR

**Comment by chrisplo** [comment]
@unclejack I'm puzzled, line 257 is nowhere near the target you are running:
```
https://github.com/chrisplo/netplugin/blob/master/Makefile#L257
```

**Comment by chrisplo** [comment]
build PR

File: scripts/netContain/build_image.sh [hunk 1]
```diff
@@ -1,74 +1,31 @@
...
 mkdir scripts/netContain/bin
 
-build_image_from_path . $BUILD_ENV_IMAGE_NAME
-if [ "$?" -ne 0 ]; then
-	echo "Failed to build the "
-	exit 1
-fi
+# it's expected that makefile targets compile-with-docker and
+# binaries-from-container have already been run
+tar c -C bin $NETPLUGIN_BINARIES | tar x -C scripts/netcontain/bin
```
Comment (unclejack) [inline scripts/netContain/build_image.sh]: This needs to be netContain.

**Comment by unclejack** [comment]
build PR

**Comment by chrisplo** [comment]
build PR

**Pull request merged**